calcom/cal.com
# fix: standardizing org subdomain url [CAL-2262] [CAL-2334]

## What does this PR do?

There were places where organization subdomain was not shown correctly, even non-org URL prefix lacked a trailing slash. This PR fixes all instances.

Fixes #10690

## Type of change


## How should this be tested?

Go to Org profile page, Org user profile page, event types page, and other places where URLs are shown to see the correct org-context treatment being applied as well as non-org-context trailing slashes being added where they were missing.

## Mandatory Tasks
